src/databases/Blueprint/avtBlueprintWriter.C

```diff
@@ -320,6 +323,10 @@ avtBlueprintWriter::WriteHeaders(const avtDatabaseMetaData *md,
                                  const vector<string> &vectors,
                                  const vector<string> &materials)
 {
+    (void)scalars; // Suppress unused parameter warning
```

Comment:
So, is there any reason to keep this around and add instead add silencing code? I mean, there are cases where this is probably best...like when you expect to be here soon again to add logic that will use these args. Otherwise, why not just remove the args?

Reply:
I don't think we can remove the args because avtBlueprintWriter inherits from avtDatabaseWriter which has this declaration:
```
    virtual void       WriteHeaders(const avtDatabaseMetaData *, 
                                    const std::vector<std::string>&scalars,
                                    const std::vector<std::string>&vectors,
                                    const std::vector<std::string>&materials) = 0;
```
So all the writers have a standard interface.

Reply:
IMO a comment explaining that the code silences a warning is easier than adding warning silencing code, since such code is compiler dependent. We would need to use macros to make comprehensive warning silencing code.